packages/dds/tree/src/test/shared-tree/summary.bench.ts

```diff
@@ -6,7 +6,7 @@
 import { strict as assert } from "assert";
 
 import { IsoBuffer } from "@fluid-internal/client-utils";
-import { BenchmarkType, benchmark } from "@fluid-tools/benchmark";
+import { BenchmarkType, benchmarkCustom, benchmark } from "@fluid-tools/benchmark";
```

Comment:
Presumably we need to wait to make these changes until we've published the new version of `benchmark`?

Reply:
https://www.npmjs.com/package/@fluid-tools/benchmark

`0.50.0` was published on Friday, and I believe the custom data concept is included in the package.